orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java

```diff
@@ -501,6 +502,18 @@ Execution retrieveOrchestrationForCorrelationId(
     });
   }
 
+  @Override
+  public @Nonnull List<Execution> retrieveBufferedExecutions() {
+    // TODO rz - This is definitely not a healthy way to do this.
```

Comment:
Already todo'd but we should definitely have an index of buffered executions before this goes live.

Reply:
+1